.gitignore

```diff
@@ -0,0 +1,10 @@
+node_modules
```

Comment:
Repository name should be changed `ckeditor-cs-on-premise-infrastructure-examples` -> `ckeditor-cs-on-premises-infrastructure-examples`.

Maybe you can also remove `examples` from the name.